virttest/utils_zchannels.py

```diff
@@ -109,6 +109,19 @@ def get_device(self, devid=None):
         else:
             return None
 
+    def get_first_dasd_in_use(self):
+        """
+        Returns the device info of the first DASD device in Use.
+
+        Requires get_info() to be called first.
+        :return: Device info
+        """
+        used = [x for x in self.devices if x[self.HEADER["Use"]] == "yes"]
+        for device in used:
+            if "3390" in device[self.HEADER["DevType"]]:
+                return device
+        return None
+
```

Comment:
I think we could do like below, that only has one loop:
for device in self.devices:
        if device[self.HEADER["Use"]] == "yes" and "3390" in device[self.HEADER["DevType"]]:
            return device
 return None

Reply:
> I think we could do like below, that only has one loop:
> 
> ```
> for device in self.devices:
>     if device[self.HEADER["Use"]] == "yes" and "3390" in device[self.HEADER["DevType"]]:
>         return device
> return None
> ```

OK, I assume you mean that's for better performance? Updated.